chad_stride.c

```diff
@@ -359,6 +374,13 @@ int main(int argc, char *argv[]) {
         /* Ncurses stuff */
         refresh();
 
+	/* display message */
+	if ( MES ) {
```

Comment:
Yo, is there any way you can have it move with chad? I think it should just be adding the current x_pos to this mesg_x :) otherwise it runs perfectly